lib/client/ipam_block_reader_writer.go

```diff
@@ -139,6 +124,20 @@ func (rw blockReaderWriter) claimNewAffineBlock(
 	return nil, noFreeBlocksError("No Free Blocks")
 }
 
+// isIPPoolInRequestedPools checks if the IP Pool that is passed in belongs to the list of IP Pools
+// that should be used for assigning IPs from.
+func isIPPoolInRequestedPools(pool net.IPNet, requestedPools []cnet.IPNet) bool {
```

Comment:
makes sense but I don't like the name `isIPNetInIPNets`, I think it's too generic and would make it hard to read. I think we should name it `isIPNetInRequestedPools`